Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/entity/deferred/response/proxy/EntityPatientDiscoveryDeferredResponseProxyWebServiceSecuredImpl.java

```diff
@@ -88,10 +90,10 @@ public MCCIIN000002UV01 processPatientDiscoveryAsyncResp(PRPAIN201306UV02 reques
                         "processPatientDiscoveryAsyncResp", securedRequest);
             } else {
                 LOG.error("Failed to call the web service (" + serviceName + ").  The URL is null.");
+                throw new WebServiceException("Could not determine URL for Patient Discovery Deferred Response endpoint");
             }
         } catch (Exception ex) {
-            LOG.error("Error: Failed to retrieve url for service: " + serviceName + " for local home community");
```

Comment:
are we losing log-error with this delete?

Reply:
The same error is being logged a few lines up.